.travis.yml

```diff
@@ -134,14 +160,14 @@ install:
 
 script:
   - mkdir build && cd build
-  - export TRAVIS_DEPENDENCY_DIR=${TRAVIS_BUILD_DIR}/dependencies
-  - export HELICS_DEPENDENCY_FLAGS="-DZeroMQ_INSTALL_PATH=${TRAVIS_DEPENDENCY_DIR}/zmq -DBOOST_INSTALL_PATH=${TRAVIS_DEPENDENCY_DIR}/boost"
+  - export HELICS_DEPENDENCY_FLAGS="-DZeroMQ_INSTALL_PATH=${CI_DEPENDENCY_DIR}/zmq -DBOOST_ROOT=${CI_DEPENDENCY_DIR}/boost"
```

Comment:
probably need to tweak this line to get the boost path to work correctly again using `BOOST_INSTALL_PATH` instead of `BOOST_ROOT`